linkedevents/settings.py

```diff
@@ -432,7 +432,7 @@ def dummy_haystack_connection_for_lang(language_code):
     HAYSTACK_CONNECTIONS.update(connection)
 
 
-BLEACH_ALLOWED_TAGS = bleach.ALLOWED_TAGS + ["p", "div", "br"]
+BLEACH_ALLOWED_TAGS = list(bleach.ALLOWED_TAGS) + ["p", "div", "br"]
```

Comment:
On a second thought, maybe more readable like this..

```suggestion
BLEACH_ALLOWED_TAGS = frozenset(list(bleach.ALLOWED_TAGS) + ["p", "div", "br"])
```

Reply:
Yeah bleach will convert the iterable into a frozenset internally. Not necessary, but IMO just good general practice  to reduce risks of accidental runtime pollution in codebase since python lists are mutable.